packages/web/src/components/search-users-modal/SearchUsersModal.tsx

```diff
@@ -155,7 +157,9 @@ export const SearchUsersModal = (props: SearchUsersModalProps) => {
             loader={<LoadingSpinner className={styles.spinner} />}
             threshold={48}
           >
-            {users.map((user) => renderUser(user, handleClose))}
```

Comment:
I know the code was already like this, but just curious if we're adding `key` props to all the users in each place we use this modal?